compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -308,11 +308,21 @@ void ExecutionPlanner::create_alloc_node_inform_vector(bool null_consts, bool nu
     {
       case luci::CircleOpcode::CONV_2D:
       {
-        auto conv = loco::must_cast<const luci::CircleConv2D *>(circle_node);
+        const auto conv = loco::must_cast<const luci::CircleConv2D *>(circle_node);
         scratchpad_size =
           null_scratchpad ? 0 : _scratchpad_helper->ComputeScratchpadSizeConv2d(conv);
         break;
       }
+      case luci::CircleOpcode::DEPTHWISE_CONV_2D:
+      {
+        const auto depthwise_conv =
+          loco::must_cast<const luci::CircleDepthwiseConv2D *>(circle_node);
+        scratchpad_size =
+          null_scratchpad
```

Comment:
hmmm, maybe move this condition outside of switch?
I suppose it will apply to every operation in condition

Reply:
Also, FYI: #8335

Reply:
Done